10.Mender-server/docs.md

```diff
@@ -8,6 +8,453 @@ github: false
 ---
 
 
+## Mender 3.6.0
+
+_Released 07.28.2023_
+
+### Statistics
+
+Processed 324 csets from 11 developers
+2 employers found
+A total of 34993 lines added, 21314 removed (delta 13679)
```

Comment:
We usually start the output at "A total of...", the others are just noise generated by the tool while working (it's an upstream tool from the Linux kernel guys).

A few more of these below as well.

Reply:
Oh, right. I force-pushed.